setup.py

```diff
@@ -35,6 +35,9 @@ def main():
         packages=find_packages(),
         long_description=read('README.md'),
         install_requires=required,
+        extras_require={
```

Comment:
👍 (Not necessary for this PR, but we should eventually list `pandas` here as well.)